sdk/core/Azure.Core/src/RequestUriBuilder.cs

```diff
@@ -24,6 +27,9 @@ public class RequestUriBuilder
 
         private string? _scheme;
 
+        /// <summary>
+        /// Provides a custom constructor for uniform resource identifiers (URIs) and modifies URIs for the <see cref="System.Uri"></see> class.
```

Comment:
Was this what you intended here?  Seems like it may have been a copy/paste mistake.